### DISCUSSION TITLE
Discrepancy between Peacock and Paraview

### DISCUSSION THREAD

**Discussion by charlotte-b8086** [idaholab/moose#26050] [Q&A General] [answered]
Hello! I am very new to MOOSE and am trying to learn how to use it by following the online 12-hour tutorial recorded in Summer 2020.

I am currently working on Step 5 but I am concerned that I have done something that is affecting my Paraview.

When I run the Step 5a from my command line using peacock (~/projects/moose/python/peacock/peacock -I step5a_transient.i) I obtain the following graphic:
<img width="1440" alt="Screenshot 2023-11-13 at 3 02 04 PM" src="https://github.com/idaholab/moose/assets/112354003/86ee3e57-11a7-483e-88dc-dc21e70737d6">
this matches what is shown in the video, as the temperature is increasing from 300 to 350. 

However, when I attempt to open the output file in Paraview using the command 'open' from my Terminal (I am using a Mac), I obtain the following as my initial and final time steps
<img width="1152" alt="Screenshot 2023-11-11 at 8 33 09 PM" src="https://github.com/idaholab/moose/assets/112354003/e331a365-7a86-4b1d-bf1d-b0d5400b3a4d">
[
<img width="1143" alt="Screenshot 2023-11-13 at 3 01 03 PM" src="https://github.com/idaholab/moose/assets/112354003/a7bdf9c9-a00b-4650-baeb-bbc53eb95fa4">
](url)

Does anyone have any explanation for why this is occurring or how to fix it? I apologize if this is a very simple question!

Thank you for your time and consideration.

**Comment by ykvishal** [comment]
When you are on last image (ie Time =1), press rescale to Data range and see what happens. 
![image](https://github.com/idaholab/moose/assets/31565781/5605ee6a-dd60-48ec-a906-a7ee48e4aa6e)

**Reply by charlotte-b8086** [reply, marked as answer]
Thank you so much! That fixed it.
